validator/src/bin/solana-test-validator.rs

```diff
@@ -218,6 +218,30 @@ fn main() {
                         If the ledger already exists then this parameter is silently ignored",
                 ),
         )
+        .arg(
+            Arg::with_name("accounts_from_dir")
+                .long("accounts-from-dir")
```

Comment:
wdyt about calling this argument `--account-dir`? I don't feel super strongly about this though

Reply:
i like it, more concise